.codespellignore

```diff
@@ -0,0 +1,5 @@
+ot
+fo
+te
+collison
```

Comment:
Why ignore "collison"? This looks like a misspelling of collision.

Reply:
Hi @MrAlias, this is a fair question. There's a person named "Patrick Collison" who probably wouldn't like to be called "Patrick Collision".

https://github.com/open-telemetry/opentelemetry-go/blob/main/exporters/jaeger/internal/third_party/thrift/LICENSE#L303